defaults/main.yml

```diff
@@ -26,6 +26,9 @@ sap_hana_deployment_license_file_name:
 # Specifies failing preequisite checks that the SAP HANA platform lifecycle management tools should ignore.
 sap_hana_deployment_hdblcm_extraargs:
 
+# set this to true to skip the deployment of HANA
+sap_hana_deployment_skip: false
```

Comment:
Same comment as the entry for this variable in the README.md file

Reply:
> Please check my comments.
> 
> I stopped the review already when I saw the changes in hana_deploy.yml. Generally speaking, I don't see the value of adding such complexity to the role to check the existing installations and show this information to the user, I suggest you move this to a different '2-day operation'-like role.
> 
I totally disagree, a role should ALWAYS be idempotent. the current role is NOT idempotent.  If you rerun this role on a system, a second time the role fails because HANA is already there and running. With my fixes the role reports back, that HANA is already running with desired SID & Instance number, installs it if it is not there, or it fails before it actually calls the installer with a proper message. The different outputs I do are helpful in debugging. Given the case the role fails, you can immediately see what's on the system.

> As I mentioned in the related issue #10 the role will already fail without hurting existing installation if the SID already exists in the system. If you want to specifically control this behaviour I'd suggest to just add this particular check and stop the role execution with a custom message.

That's what I do. But I stop the role without an error, if the correct stuff is running

Reply:
Removed